hadoop-ozone/dist/src/main/compose/ozone/test.sh

```diff
@@ -28,6 +28,8 @@ source "$COMPOSE_DIR/../testlib.sh"
 
 start_docker_env
 
+execute_robot_test scm httpfs
```

Comment:
Should we try out the interface after the CLI tests at the end, when we already know that everything is working fine in Ozone to avoid false positives in httpfs gw tests?

Reply:
Good idea, I left it there because of the testing in my own environment. Moved the executing of the HttpFS tests after the CLI tests.